tools/js-sdk-release-tools/src/common/rushUtils.ts

```diff
@@ -112,6 +113,20 @@ export async function tryBuildSamples(packageDirectory: string, rushxScript: str
     }
 }
 
+// no exception will be thrown, since we don't want it stop sdk generation. sdk author will need to resolve the failure
+export async function tryFormatSamples(packageDirectory: string) {
```

Comment:
I think it should be required, and it can fail the sdk automation, otherwise, release pipeline will be failed anyway, @kazrael2119 is that true?

Reply:
normally, I think format should not fail, it only modifies the construction

Reply:
@skywing918 please make it required